### PR TITLE
Support meta information on service

### PR DESCRIPTION
**Pull request by krjensen**

### CODE REVIEW

**Comment by solarkennedy** [comment]
Can you pull/rebase against master please?

File: manifests/service.pp [hunk 1]
```diff
@@ -43,6 +43,7 @@
   $service_name        = $title,
   $tags                = [],
   $token               = undef,
+  $meta                = {},
```
[the remark this answers is not in the document]
Comment (krjensen) [reply manifests/service.pp]: Not sure I get you? Its a `{}` that per definition is a Hash? Just like the $tags are a array?

[the remark this answers is not in the document]
Comment (bastelfreak) [reply manifests/service.pp]: the default datatype is `Hash` in this case, but users could provide every other datatype as well. You can prefix the parameter with an explicit datatype, to enforce it. That would be:
```puppet
Hash $meta = {},
```

**Comment by krjensen** [comment]
I think this is a duplicate of https://github.com/solarkennedy/puppet-consul/pull/436 ?
Should we close this on and work on 436?

**Comment by jardleex** [comment]
I think this can get closed due to #466

**Pull request closed without merging**